packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/components/LabelType.java

```diff
@@ -35,4 +44,14 @@ public enum LabelType {
     public String getValue() {
         return this.value;
     }
+    
+    static LabelType valueOfByValue(String value) throws IllegalArgumentException {
```

Comment:
Why `throws IllegalArgumentException`? You could also name the method something like `fromValue` to have something like `LabelType.fromValue("...")`

Reply:
Done, the `IllegalArgumentException` was remaining from an old version of the PR.